setup.py

```diff
@@ -49,7 +49,7 @@ def read(fname):
 
       extras_require={
           'test': ['tox', 'flake8', 'pytest', 'pytest-cov', 'pytest-xdist',
-                   'mock', 'tensorflow>=1.3.0', 'contextlib2', 'awslogs', 'pandas']},
+                   'mock', 'tensorflow>=1.3.0', 'protobuf3-to-dict', 'contextlib2', 'awslogs', 'pandas']},
```

Comment:
Could you move `protobuf3-to-dict` to `install_requires` (and add a version)?

Reply:
Done, thank you.